adcm_aio_client/core/objects/_common.py

```diff
@@ -48,11 +47,11 @@ class WithMaintenanceMode(WithProtectedRequester, AwareOfOwnPath):
     @async_cached_property
     async def maintenance_mode(self: Self) -> MaintenanceMode:
         maintenance_mode = MaintenanceMode(self._data["maintenanceMode"], self._requester, self.get_own_path())  # pyright: ignore[reportAttributeAccessIssue]
-        self._data["maintenanceMode"] = maintenance_mode.value  # pyright: ignore[reportAttributeAccessIssue]
+        self._data["maintenanceMode"] = maintenance_mode  # pyright: ignore[reportAttributeAccessIssue]
```

Comment:
it's not part of data btw

data is raw data from API and shouldn't be changed in that way